src/dali/plugin/input/csv/blockfi.py

```diff
@@ -133,9 +177,98 @@ def load(self) -> List[AbstractTransaction]:
                             crypto_received=Keyword.UNKNOWN.value,
                         )
                     )
-                elif line[self.__TYPE_INDEX] == _WITHDRAWAL_FEE:
+                elif entry_type == _ACH_DEPOSIT:
+                    last_withdrawal_fee = None
+                    result.append(
+                        InTransaction(
+                            plugin=self.__BLOCKFI,
+                            unique_id=Keyword.UNKNOWN.value,
+                            raw_data=raw_data,
+                            timestamp=f"{line[self.__TIMESTAMP_INDEX]} -00:00",
+                            asset=line[self.__CURRENCY_INDEX],
+                            exchange=self.__BLOCKFI,
+                            holder=self.account_holder,
+                            transaction_type="Buy",
+                            spot_price=Keyword.UNKNOWN.value,
+                            crypto_in=line[self.__AMOUNT_INDEX],
+                            fiat_fee="0",
+                            notes="Ach deposit",
```

Comment:
If this is a crypto buy (not a fiat movement) I would remove the notes assignment, because I imagine "ACH" would be confusing to the final user.

Reply:
ACH deposit is a fiat movement, not a crypto buy, it's just that blockfi tracks it as GUSD token.

Reply:
Ok.